build.gradle

```diff
@@ -71,3 +74,65 @@ dependencies {
         //implementation 'org.springframework:spring-web:5.0.2.RELEASE'
     }
 }
+
+def classFilesArray = []
+def execFilesArray = []
+
+task copyExecFilesAndJavaClassFiles {
+    subprojects.forEach { subproject -> 
+        def execFile = new File("${subproject.buildDir}/jacoco/jacoco.exec")
+        if(execFile.exists()){
+            execFilesArray.push(execFile)
+            fileTree("${subproject.buildDir}/classes").matching{
+                exclude '**/test/*'
+                exclude '**/module-info.class'
+                include '**/*.class'
+            }.each{file->classFilesArray .push(file)}
```

Comment:
```suggestion
            }.each{file->classFilesArray.push(file)}
```

Reply:
Please update formatting in other places as well

Reply:
commit id is 24da11be946e9559251b50d53f5364cc685a25f8